Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -1263,6 +1263,13 @@ static bool exceedsRenderTreeSizeSizeThreshold(uint64_t thresholdSize, uint64_t
 
 #endif
 
+#if ENABLE(GAMEPAD)
+void WebPageProxy::platformGamepadsRecentlyAccessed(GamepadsRecentlyAccessed gamepadsRecentlyAccessed)
+{
+    [cocoaView().get() _setGamepadsRecentlyAccessed:(gamepadsRecentlyAccessed == GamepadsRecentlyAccessed::No) ? NO : YES];
```

Comment:
Access to the underlying view is normally done via `PageClient`.

(also, `.get()` is unnecessary)